versioning/BRM/sessionmanagerserver.h

```diff
@@ -209,15 +210,15 @@ class SessionManagerServer
    */
   uint32_t getUnique32()
   {
-    return atomicops::atomicInc(&unique32);
+    return unique32.fetch_add(1, std::memory_order_relaxed) + 1;
```

Comment:
I am missing why do we call fetch_add and then `+1` on the result? Doesn't this operation adds 2 instead of increment by one?